### PR TITLE
Add oxwu 3.0.0

### PR DESCRIPTION
**Pull request by Haraguroicha**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/oxwu.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# typed: false
+# frozen_string_literal: true
+
+cask "oxwu" do
```
Comment (SMillerDev) [inline Casks/oxwu.rb:4]: These aren't needed
```suggestion
cask "oxwu" do
```

Comment (Haraguroicha) [reply Casks/oxwu.rb]: removed them and amended

**Comment by bevanjkay** [comment]
@Haraguroicha Regarding the cask token, I think `ox-wake-up` might be better? Is this what the application would be known as in English?

**Comment by Haraguroicha** [comment]
@bevanjkay `oxwu` is regarding that application original named inside, but translate from display name is okay for using `ox-wake-up`

here is the reason of why I'm choosing `oxwu` instead of `ox-wake-up`, because application package.json is wrote by following
image

**Review by miccal — APPROVED** [review]

**Comment by miccal** [comment]
Thank you @Haraguroicha.

**Pull request merged**